css/index.css

```diff
@@ -1,3 +1,461 @@
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 27px;
+}
+                /* --- NAVIGATION starts here --- */
+nav {
+    display: flex;
+    justify-content: space-around;
+    width: 70%;
+    margin-top: 4px;
+    margin-left: 42px;
+    margin-right: 26px;
+}
+
+nav a {
+    color: gray;
+    font-size: 16px;
+    font-weight: bold;
+}
+                /* --- NAVIGATION ends here --- */
+
+.header-top a.home-page {
+    width: 30%;
+    align-self: flex-end;
+    padding-left: 35px;
+}
+        /* ---- HEADER-TOP ends here ----- */
+
+
+        /* ---- HEADER-BOTTOM starts here ----- */
+.header-bottom {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 57px;
+}
+
+.header-bottom > div {
+    flex: 1;
+}
+
+.title {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    padding-left: 121px;
+    padding-top: 12px;
+}
+
+img.code {
+    margin-top: 14px;
+    margin-left: 2px;
+}
+        /* ---- HEADER-BOTTOM ends here ----- */
+/* ------------- HEADER ends here ------------- */
+
+
+hr.one {
+    margin-top: 73px;
+    border: 1px solid black;
+    margin-left: 74px;
+    width: 880px;
+}
+
+/* ------------ CONTENT starts here ------------ */
+.content {
+    width: 100%;
+    margin-left: 74px;
+    margin-top: 36px;
+    margin-right: 72px;
+}
+
+        /* ---- CONTENT-TOP starts here ---- */
+.content-top {
+    width: 85%;
+    display: flex;
+    justify-content: flex-start;
+    margin-bottom: 30px;
+}
+
+.content-top > div {
+    flex: 1;
+}
+
+.content-top > div:nth-child(2) {
+    margin-left: 9px;
+}
+
+.content-top > div > h2 {
+    margin-bottom: 16px;
+}
+
+.content-top > div > p {
+    line-height: 1.13em;
+}
+        /* ---- CONTENT-TOP ends here ---- */
+
+aside {
+    margin-left: 2px;
+    margin-bottom: 29px;
+}
+
+        /* ---- CONTENT-BOTTOM starts here ---- */
+.content-bottom {
+    width: 86%;
+    display: flex;
+    justify-content: flex-start;
+}
+
+.content-bottom > div {
+    flex: 1;
+    padding-right: 20px;
+}
+
+.content-bottom > div > h2 {
+    margin-bottom: 16px;
+}
+
+.content-bottom > div > p {
+    line-height: 1.13em;
+}
+        /* ---- CONTENT-BOTTOM ends here ---- */
+/* ------------ CONTENT ends here ------------ */
+
+hr.two {
+    margin-top: 35px;
+    border: 1px solid black;
+    margin-left: 74px;
+    width: 880px;
+}
+
+/* ---------- FOOTER starts here ---------- */
+footer {
+    width: 100%;
+    margin-left: 74px;
+    margin-top: 35px;
+}
+
+footer h2 {
+	margin-bottom: 15px;
+}
+
+.address {
+	margin-bottom: 15px;
+}
+
+.phone {
+	margin-bottom: 15px;
+}
+
+.copyright {
+    margin-top: 40px;
+    margin-left: 350px;
+}
+/* ---------- FOOTER ends here ---------- */
+
+
+
+
+/* ===================== TABLET version ======================== */
+
+@media (max-width: 800px) {
+    /* ---- HEADER ---- */
+    header {
```

Comment:
I recommend you start your media queries with your container.. setting the width on that should make a lot of stuff fall in place, specially when you're using percentages on child element widths.